modules/gitops-release-notes-1-5-5.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-5-5_{context}"]
+= Release notes for {gitops-title} 1.5.5
+
+{gitops-title} 1.5.5 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="new-features-1-5-5_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the bundled Argo CD has been updated to version 2.3.7.
+
+[id="fixed-issues-1-5-5_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCC's were in place in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]
```

Comment:
```suggestion
* Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCCs were in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]
```

Reply:
IBM SG: "Use an apostrophe and the letter s ('s) to form the plural of single letters." (but not multiple letters) For example: "Correct: The basic learning requirements for any industry are often known as the ABCs."

Reply:
It looks like this change hasn't been made yet. How about:
```
... when more restrictive SCCs were present in the cluster....
```